test/FunctionalTestUtils/TelemetryTestsBase.cs

```diff
@@ -48,5 +52,42 @@ public void ValidateBasicException(InProcessServer server, string requestPath, E
             Assert.NotEmpty(actual.Context.Operation.Name);
             Assert.NotEmpty(actual.Context.Operation.Id);
         }
+
+        public void ValidateBasicDependency(InProcessServer server, string requestPath, DependencyTelemetry expected)
+        {
+
+#if dnx451 
```

Comment:
The actual tests are completely wrapped around the condition (which will not account to the passed count). This is a function that is being used by the tests. It also makes sense to wrap around this. Will do that.